src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,477 @@
+        }
+
+        @DisplayName("메뉴 가격 완료")
+        @Test
+        void changePriceTest5() {
+
+            // given
+            final UUID menuId = UUID.randomUUID();
+            final Product product = ProductFixture.createProductWithPrice(500);
+            final MenuProduct menuProduct = MenuProductFixture.createMenuProduct(product, 1);
+            final Menu request = MenuFixture.createMenuWithMenuProductsAndPrice(List.of(menuProduct), 400);
+
+            given(menuRepository.findById(any()))
+                    .willReturn(Optional.of(request));
+
+            // when
+            Menu actual = menuService.changePrice(menuId, request);
+
+            // then
+            assertThat(actual).isNotNull();
+            assertThat(actual.getPrice()).isEqualTo(request.getPrice());
+        }
+    }
+
+    @DisplayName("메뉴 전시")
+    @Nested
+    class DisplayTestGroup {
+
+        @DisplayName("등록된 메뉴가 아니면 예외 발생")
+        @Test
+        void displayTest1() {
+
+            // given
+            final UUID menuId = UUID.randomUUID();
+            final Menu menu = MenuFixture.createMenu();
+
+            given(menuRepository.findById(any()))
+                    .willReturn(Optional.empty());
+
+            // when + then
+            assertThatExceptionOfType(NoSuchElementException.class)
+                    .isThrownBy(() -> menuService.display(menuId));
+        }
+
+        @DisplayName("메뉴 가격이 상품의 가격과 수량을 곱한 가격과 같거나 높지 않으면 예외 발생")
+        @Test
+        void displayTest2() {
+
+            // given
+            final UUID menuId = UUID.randomUUID();
+            final Product product = ProductFixture.createProductWithPrice(500);
+            final MenuProduct menuProduct = MenuProductFixture.createMenuProduct(product, 1);
+            final Menu menu = MenuFixture.createMenuWithMenuProductsAndPrice(List.of(menuProduct), 700);
+
+            given(menuRepository.findById(any()))
+                    .willReturn(Optional.of(menu));
+
+            // when + then
+            assertThatExceptionOfType(IllegalStateException.class)
+                    .isThrownBy(() -> menuService.display(menuId));
+        }
+
+        @DisplayName("메뉴 전시 완료")
+        @Test
+        void displayTest3() {
+
+            // given
+            final UUID menuId = UUID.randomUUID();
+            final Product product = ProductFixture.createProductWithPrice(500);
+            final MenuProduct menuProduct = MenuProductFixture.createMenuProduct(product, 1);
+            final Menu menu = MenuFixture.createMenuWithMenuProductsAndPriceAndDisplayed(List.of(menuProduct), 500, false);
+
+            given(menuRepository.findById(any()))
+                    .willReturn(Optional.of(menu));
+
+            // when
+            Menu actual = menuService.display(menuId);
+
+            // then
+            assertThat(actual).isNotNull();
+            assertThat(actual.isDisplayed()).isTrue();
```

Comment:
단언문이 여러개라면 `Assertions#assertAll` 를 사용해 보는 것은 어떨까요? 
어떤 차이가 있을까요? 
```suggestion
            assertAll(
                () -> assertThat(actual).isNotNull(),
                () -> assertThat(actual.getPrice()).isEqualTo(request.getPrice())
            );
```

Reply:
> 단언문이 여러개라면 `Assertions#assertAll` 를 사용해 보는 것은 어떨까요?

비동기 스트림으로 평가를 동시에 진행한다는 측면에서 차이가 있는 것 같습니다.
다만, NotNull 을 먼저 체크를 하지 않고 사용 할 경우 NPE 이슈도 있는 것 같습니다.